synapse/storage/schema/delta/53/drop_sent_transactions.sql

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2015, 2016 OpenMarket Ltd
+/* Copyright 2018 New Vector Ltd
```

Comment:
New Vector didn't write it, so this isn't really true.

You wrote it (all six words of it), so it's copyright yourself (with a license for us to use it via the DCO), either implicitly or explicitly. So either: `Copyright 2018 Aaron Raimist`, or just omit it.